tests/testLib.ml

```diff
@@ -7,6 +7,9 @@ let ctez_addr = Ligo.address_of_string "ctez_addr"
 let oracle_addr = Ligo.address_of_string "oracle_addr"
 
 let qcheck_to_ounit t = OUnit.ounit2_of_ounit1 @@ QCheck_ounit.to_ounit_test t
+let show_key_hash_option (key_hash: Ligo.key_hash option) = match key_hash with
+  | None -> "None"
+  | Some kh -> "Some " ^ (Ligo.string_of_key_hash kh)
```

Comment:
You can also avoid the boilerplate if you approach this the same way it's approached for the other basic types. See for example `tez option` below:
```ocaml
type tez_option = Ligo.tez option [@@deriving show]
let assert_tez_option_equal ~expected ~real = OUnit2.assert_equal ~printer:show_tez_option expected real
```